### PR TITLE
PFM dashboard screen, Category scree, transaction history screen

### PR DESCRIPTION
**Pull request by Salman1019864**
### Why

Please give a short description why these code changes are being made, by replacing this line.

Link to Jira ticket: <place here>

https://101digital.atlassian.net/browse/ADBP-2100
https://101digital.atlassian.net/browse/ADBP-2099
https://101digital.atlassian.net/browse/ADBP-2098

### ToDo

- [ ]
- [ ]
- [ ]

If this is a `Draft:` or `WIP:` PR, please describe still needed actions, else delete section

### CODE REVIEW

File: src/components/adb-personal-financial-management-dashboard/index.tsx
Comment (sharmal33) [inline src/components/adb-personal-financial-management-dashboard/index.tsx:1]: why this value are hardcoded.

plz use theme colors

File: src/components/adb-pfa-category-deep-dive/index.tsx
Comment (sharmal33) [inline src/components/adb-pfa-category-deep-dive/index.tsx:1]: plz use theme colors

File: src/components/adb-pfa-transaction-history/index.tsx
Comment (sharmal33) [inline src/components/adb-pfa-transaction-history/index.tsx:1]: plz move all static content into translations file

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/adb-pfa-category-deep-dive/index.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.content}>
+
+        <View style={{marginTop: 0, paddingHorizontal: 22,}}>
+          <FlatList
+            horizontal={true}
+            showsHorizontalScrollIndicator={false}
+            data={['Food', 'Utilities', 'Transfer', 'Others' ]}
```
Comment (sharmal33) [inline src/components/adb-pfa-category-deep-dive/index.tsx:39]: plz move these to const file -> src/helpers/screen-utils

File: src/components/adb-pfa-category-deep-dive/index.tsx
Comment (sharmal33) [inline src/components/adb-pfa-category-deep-dive/index.tsx:1]: move all static content to translations file

**Review by sharmal33 — CHANGES_REQUESTED** [review]

**Comment by Salman1019864** [comment]
all static content to translations file

this pr also have some changes after text is moved 
PR
https://github.com/101digital/aeon-app/pull/105

**Pull request merged**